bucket/btop-lhm.json

```diff
@@ -0,0 +1,38 @@
+{
+    "version": "1.0.2",
+    "description": "A monitor of resources, btop++ for windows",
```

Comment:
Might be good to also note the differences between the two manifests in the description field (like most manifests here):

```suggestion
    "description": "A monitor of resources, btop++ for windows - version with full monitoring capabilities using LibreHardwareMonitor",
```

Reply:
This is already in the notes

Reply:
It would be helpful for people searching though https://scoop.sh/, see example:
![image](https://user-images.githubusercontent.com/75471378/189525927-d81b43d5-daec-4940-acae-341cc069d892.png)